python/ray/tune/execution/experiment_state.py

```diff
@@ -34,45 +33,22 @@
 
 @dataclass
 class _ResumeConfig:
-    resume_unfinished: bool = True
-    resume_errored: bool = False
-    restart_errored: bool = False
-
-
-def _resume_str_to_config(resume_str: str) -> Tuple[str, _ResumeConfig]:
-    if resume_str is True:
-        resume_str = "LOCAL"
-    elif resume_str == "ERRORED_ONLY":
-        warnings.warn(
-            "Passing `resume='ERRORED_ONLY'` to tune.run() is deprecated and "
-            "will be removed in the future. Please pass e.g. "
-            "`resume='LOCAL+RESTART_ERRORED_ONLY'` instead."
-        )
-        resume_str = "LOCAL+RESTART_ERRORED_ONLY"
-
-    # Parse resume string, e.g. AUTO+ERRORED
-    resume_config = _ResumeConfig()
-    resume_settings = resume_str.split("+")
-    resume_str = resume_settings[0]
-
-    for setting in resume_settings:
-        if setting == "ERRORED":
-            resume_config.resume_errored = True
-        elif setting == "RESTART_ERRORED":
-            resume_config.restart_errored = True
-        elif setting == "ERRORED_ONLY":
-            resume_config.resume_unfinished = False
-            resume_config.restart_errored = False
-            resume_config.resume_errored = True
-        elif setting == "RESTART_ERRORED_ONLY":
-            resume_config.resume_unfinished = False
-            resume_config.restart_errored = True
-            resume_config.resume_errored = False
-
-    assert resume_str in VALID_RESUME_TYPES, "resume={} is not one of {}".format(
-        resume_str, VALID_RESUME_TYPES
-    )
-    return resume_str, resume_config
+    class ResumeType(Enum):
+        """An enumeration to define resume types for various trial states.
+
+        Members:
+            RESUME: Resume from the latest checkpoint.
+            RESTART: Restart from the beginning (with no checkpoint).
+            IGNORE: Ignore this trial.
```

Comment:
nit: Upon first read I wasn't sure what "ignore" meant, but not sure off the top of my head if there's a clearer word we can use here.

Reply:
Updated to `SKIP`